collectors/ebpf.plugin/README.md

```diff
@@ -39,815 +35,826 @@ The following two features from the Linux kernel are used by Netdata to run eBPF
-- `shmdt`: Runs when [`shmdt`](https://man7.org/linux/man-pages/man2/shmat.2.html) is called.
-- `shmctl`: Runs when [`shmctl`](https://man7.org/linux/man-pages/man2/shmctl.2.html) is called.
+```conf
+[global]
+   cgroups = yes
+```
 
-### Memory
+If you do not need to monitor specific metrics for your `cgroups`, you can enable `cgroups` inside
+`ebpf.d.conf`, and then disable the plugin for a specific `thread` by following the steps in the
+[Configuration](#configuration) section.
 
-In the memory submenu the eBPF plugin creates two submenus **page cache** and **synchronization** with the following
-organization:
+#### Integration Dashboard Elements
 
-* Page Cache
-    * Page cache ratio
-    * Dirty pages
-    * Page cache hits
-    * Page cache misses
-* Synchronization
-    * File sync
-    * Memory map sync
-    * File system sync
-    * File range sync
+When an integration is enabled, your dashboard will also show the following cgroups and apps charts using low-level
+Linux metrics:
 
-#### Page cache hits
+> Note: The parenthetical accompanying each bulleted item provides the chart name.
 
-When the processor needs to read or write a location in main memory, it checks for a corresponding entry in the page cache.
- If the entry is there, a page cache hit has occurred and the read is from the cache.
+- mem
+    - Number of processes killed due out of memory. (`oomkills`)
+- process
+    - Number of processes created with `do_fork`. (`process_create`)
+    - Number of threads created with `do_fork` or `clone (2)`, depending on your system's kernel
+      version. (`thread_create`)
+    - Number of times that a process called `do_exit`. (`task_exit`)
+    - Number of times that a process called `release_task`. (`task_close`)
+    - Number of times that an error happened to create thread or process. (`task_error`)
+- swap
+    - Number of calls to `swap_readpage`. (`swap_read_call`)
+    - Number of calls to `swap_writepage`. (`swap_write_call`)
+- network
+    - Number of bytes sent. (`total_bandwidth_sent`)
+    - Number of bytes received. (`total_bandwidth_recv`)
+    - Number of calls to `tcp_sendmsg`. (`bandwidth_tcp_send`)
+    - Number of calls to `tcp_cleanup_rbuf`. (`bandwidth_tcp_recv`)
+    - Number of calls to `tcp_retransmit_skb`. (`bandwidth_tcp_retransmit`)
+    - Number of calls to `udp_sendmsg`. (`bandwidth_udp_send`)
+    - Number of calls to `udp_recvmsg`. (`bandwidth_udp_recv`)
+- file access
+    - Number of calls to open files. (`file_open`)
+    - Number of calls to open files that returned errors. (`open_error`)
+    - Number of files closed. (`file_closed`)
+    - Number of calls to close files that returned errors. (`file_error_closed`)
+- vfs
+    - Number of calls to `vfs_unlink`. (`file_deleted`)
+    - Number of calls to `vfs_write`. (`vfs_write_call`)
+    - Number of calls to write a file that returned errors. (`vfs_write_error`)
+    - Number of calls to `vfs_read`. (`vfs_read_call`)
```

Comment:
A suggestion, considering how we show data. It is better to move `vfs_read_error`for the line `179`.

Reply:
Agreed!